client/my-sites/email/email-management/home/email-plan-header.jsx

```diff
@@ -38,9 +38,11 @@ const EmailPlanHeader = ( {
 				<div>
 					<h2>{ domain.name }</h2>
 
-					<span className="email-plan-header__status">
-						<MaterialIcon icon={ icon } /> { text }
-					</span>
+					{ emailAccount?.warnings?.[ 0 ] && (
+						<span className="email-plan-header__status">
+							<MaterialIcon icon={ icon } /> { text }
+						</span>
+					) }
```

Comment:
I think this is the wrong approach -- I think we need to update `resolveEmailPlanStatus()` to remove the logic in that function which returns an error when `hasPendingGSuiteUsers( domain )` is true.